Sources/DiscoverKit/View/ErrorView.swift

```diff
@@ -9,12 +9,23 @@ struct ErrorView: View {
     let error: Error
 
     var body: some View {
-        // TODO: add implementation
         // TODO: add proper error logging
-        Text("There was an error \(error.localizedDescription)")
-            .refreshable {
-                viewModel.load()
+        ScrollView {
+            ZStack {
+                Color(.mosoLayerColor1).ignoresSafeArea()
+                VStack(alignment: .leading, spacing: 8) {
+                    Text("Sorry, Something went wrong")
+                        .font(.title)
+                    Text("We are all about a healthy internet but sometimes things break.")
```

Comment:
based on Figma File:
```suggestion
                    Text("We're all about a healthy internet but sometimes things break.")
```

not necessarily needed for this PR, should we think about moving hard coded strings into its own file?

Reply:
Good point! I'm hoping to add proper localization soon, which will solve this issue.